### PR TITLE
Admin users

### PR DESCRIPTION
**Pull request by jpc20**
Admins can now see all users.
Admins can also view all user profiles.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
   namespace :admin do
     get '/', to: 'dashboard#index'
-    get '/users', to: 'users#index'
+    resources :users, only: [:index, :show]
```
Comment (Ashkanthegreat) [inline config/routes.rb:13]: Good use of resources

File: app/views/admin/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1><%= @user.name %></h1>
+<p> <%= @user.email %></p>
+<p> <%= @user.address %></p>
```
Comment (Lithnotep) [inline app/views/admin/users/show.html.erb:3]: this is what i needed for US31

File: config/routes.rb (hunk 1 above)
Comment (janegreene) [inline config/routes.rb:13]: Very sleek

Comment (Lithnotep) [inline config/routes.rb:13]: nice clean up of resources

Comment (jpc20) [reply config/routes.rb:13]: Thanks Ash!

File: app/views/admin/users/show.html.erb (hunk 2 above)
Comment (jpc20) [reply app/views/admin/users/show.html.erb:3]: Right on!

File: config/routes.rb (hunk 1 above)
Comment (jpc20) [reply config/routes.rb:13]: Thanks Jane!

**Pull request merged**